composer.json

```diff
@@ -40,7 +40,9 @@
         "pda/pheanstalk": "~3.0",
         "flow/jsonpath": "~0.2",
         "league/factory-muffin": "^3.0",
-        "league/factory-muffin-faker": "^1.0"
+        "league/factory-muffin-faker": "^1.0",
+        "ext-mongodb": "*",
```

Comment:
nah, codeception by itself does not require mongodb. I will remove this constraint